### PR TITLE
Add: Pass minComplexity and maxComplexity as options params

### PR DESCRIPTION
**Pull request by greatbigsled**

### CODE REVIEW

**Comment by podkot** [comment]
@shifteee jfyi

File: jquery.complexify.js [hunk 1]
```diff
@@ -166,11 +168,12 @@
 
         // Use natural log to produce linear scale
         complexity = Math.log(Math.pow(complexity, password.length)) * (1/options.strengthScaleFactor);
+        console.log("real complexity ", complexity);
```
Comment (podkot) [inline jquery.complexify.js]: rm console.log from diff pls

Comment (greatbigsled) [reply jquery.complexify.js]: Done. Removed and rebuilt min file

**Comment by podkot** [comment]
thanks, landed

could you provide a PR to the upstream too?

**Comment by greatbigsled** [comment]
Done

**Pull request merged**